contributing/creating-great-first-issues.md

```diff
@@ -0,0 +1,22 @@
+# Creating Great First Issues
+
+The AMP Project welcomes new contributors and we want to make it as easy as possible for them to contribute.  For many new contributors (who may not have open source/Git/AMP/etc. experience) it can be difficult to figure out how to get started.
+
+To help these new contributors get oriented we curate [Great First Issues](https://github.com/ampproject/amphtml/issues?q=is%3Aopen+is%3Aissue+label%3A%22Great+First+Issues%22).  A Great First Issue is a starter issue that a new contributor can use to get comfortable contributing to the AMP Project.
```

Comment:
We spoke in person and decided on the approach Malte suggested.  I changed the milestone to be called Great First Issues, and will change the label to GFI Candidates once this PR goes in.